internal/config/configfile.go

```diff
@@ -465,6 +468,13 @@ func (c Config) Validate() (err error) {
 		}
 	}
 
+	if err == nil {
+		missingKeyValues := []string{"error", "zero"}
+		if !slices.Contains(missingKeyValues, c.MissingKey) {
+			err = fmt.Errorf("not allowed value for the 'missing-key' flag: %s. Allowed values: %s", c.MissingKey, strings.Join(missingKeyValues, ","))
```

Comment:
the tests are failing currently because of this check - the value may also be empty, or `default`:

```suggestion
		missingKeyValues := []string{"", "error", "zero", "default"}
		if !slices.Contains(missingKeyValues, c.MissingKey) {
			err = fmt.Errorf("invalid value for 'missingKey': %q. Must be one of: %v", c.MissingKey, missingKeyValues))
```

Reply:
Done